### PR TITLE
Add github contribution slides

### PR DESCRIPTION
**Pull request by cmonjeau**
Hello hello,

here a little pull request on the pull requests with github..... it's like the inception.

@abretaud have already read the slides so if there are mistakes, it's his fault. Shame on him.

Thanks,
Cyril

### CODE REVIEW

File: Dev-Corner/slides/index.html [hunk 1]
```diff
@@ -82,6 +82,8 @@
                         - [Tool development and integration into Galaxy](tool_integration.html)
                         - [Data source integration](data_source_integration.html)
                         - [Visualizations in Galaxy](visualizations.html)
+                        - [Contribute with GitHub](github_contribution.html)
```
Comment (abretaud) [inline Dev-Corner/slides/index.html]: Contributing héhé

Comment (cmonjeau) [inline Dev-Corner/slides/index.html]: héhéhé

**Comment by abretaud** [comment]
Oh, you're red @cmonjeau 
Anyway, we made a small session today with these new slides + http://bgruening.github.io/training-material/Dev-Corner/slides/tool_integration.html and it went quite well I think. We found some small things to fix in the tool_integration slides, we'll probably make a PR on this some day

**Comment by bgruening** [comment]
Nice work!!!

**Comment by abretaud** [comment]
Hum, just a question about this training: is there a way to use a locally built bioconda dependency when doing a planemo test? Or is it required that the conda package is pushed to the bioconda channel before using it in a galaxy tool?
I remembered seeing this do-everything-locally idea somewhere, but can't remember if this is implemented or not.

**Comment by cmonjeau** [comment]
Hellooo,

there are two checks failed:

[404] https://github.com/username/training-material -> it's a demo url for the example so it's totally normal.

[404] http://bgruening.github.io/training-material/Dev-Corner/slides/github_contribution.html -> it's the future link for the presentation. 

Do I have to fix something to pass tests?

**Comment by bgruening** [comment]
For the first ... we currently have no mechanism in place to skip URLs from the check. But if we merge this with the link all our testing will not work, because it will always fail on this URL. I need to find a solution howto mark a URL as intentionally broken.

Can you comment the second one as long as the slides are not ready?

**Comment by bgruening** [comment]
@cmonjeau we are using `vl` to check URLs. Can you try locally and check this file with something like `vl README.md -w https://github.com/username/training-material` -w is the whitelist parameter.
I'm just worried that this will ignore all other github URLs as well not only the username one ...

**Comment by cmonjeau** [comment]
After some test, I'm afraid that the -w option doesn't accept a specific url, but only a domain name. 

Below two tests on the readme.md file: 

cmonjeau@cmonjeau:~/training-material$ vl Dev-Corner/readme.md -w bgruening.github.io

[200] https://github.com/bgruening/training-material/issues/77
[200] https://github.com/bgruening/training-material/issues/78
[200] https://github.com/bgruening/training-material/issues/74
[200] https://github.com/bgruening/training-material/issues/76
Total Links Parsed 11
Total Errors 0
Total Exceptions 1
Total Dupes 1
Total whitelisted 5
Total static 0

cmonjeau@cmonjeau:~/training-material$ vl Dev-Corner/readme.md -w http://bgruening.github.io/training-material/Dev-Corner/slides/github_contribution.html

[200] http://bgruening.github.io/training-material/Dev-Corner/slides/
[200] http://bgruening.github.io/training-material/Dev-Corner/slides/visualizations.html
[404] http://bgruening.github.io/training-material/Dev-Corner/slides/github_contribution.html
[200] http://bgruening.github.io/training-material/Dev-Corner/slides/tool_integration.html
[200] http://bgruening.github.io/training-material/Dev-Corner/slides/data_source_integration.html
[200] https://github.com/bgruening/training-material/issues/77
[200] https://github.com/bgruening/training-material/issues/74
[200] https://github.com/bgruening/training-material/issues/78
[200] https://github.com/bgruening/training-material/issues/76

Failed URLs:
[404] http://bgruening.github.io/training-material/Dev-Corner/slides/github_contribution.html
Total Links Parsed 11
Total Errors 1
Total Exceptions 1
Total Dupes 1
Total whitelisted 0
Total static 0

**Comment by cmonjeau** [comment]
And vl reads the links in comment:



Failed URLs:
[404] http://bgruening.github.io/training-material/Dev-Corner/slides/github_contribution.html

**Comment by bgruening** [comment]
Thats a bummer. Can you describe the URL in text and work around it somehow. I think checking URLs is in general a good idea, but we need to find a solution to skip certain ones at some point.

**Comment by cmonjeau** [comment]
Ooooo it's green!!! @abretaud @bgruening

**Comment by abretaud** [comment]
Miracle!

**Comment by cmonjeau** [comment]
@abretaud I can't take my eyes off this picture.

**Comment by bgruening** [comment]
A Miracle indeed :)

**Pull request merged**